sycl/doc/extensions/proposed/sycl_ext_oneapi_prefetch.asciidoc

```diff
@@ -190,6 +193,16 @@ void prefetch(T* ptr, Properties properties = {});
 template <typename T, typename Properties = ext::oneapi::experimental::properties<>>
 void prefetch(T* ptr, size_t count, Properties properties = {});
 
+template <typename T,
+          typename PtrProperties,
+          typename ArgProperties = ext::oneapi::experimental::properties<>>
```

Comment:
```suggestion
          typename ArgProperties = empty_properties_t>
```

We decided recently to use this type alias to name the type of an empty property list.  I think there is no need for all the namespaces since the prefetch functions are also defined in that same namepace.